docker/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3.6'
+services:
+  elasticsearch-01:
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.16.2
+    container_name: elasticsearch-01
+    environment:
+      - node.name=elasticsearch-01
+      - cluster.name=ctautocomplete-es
+      - cluster.initial_master_nodes=elasticsearch-01
+      - bootstrap.memory_lock=true
+      - "ES_JAVA_OPTS=-Xms1g -Xmx1g"
```

Comment:
If we have a beefy VM we might want to bump the memory limit? Can it come from a config?  1GB should be enough to fit everything in RAM, but will be future proof ;D

Reply:
Yea defaulted to 2gb - 2gb for now. Should be more than enough
🔧